app/models/rbac.rb

```diff
@@ -323,6 +323,11 @@ def self.get_user_info(user, userid, miq_group, miq_group_id)
   end
 
   def self.filtered(objects, options = {})
+    if objects.nil?
+      Vmdb::Deprecation.deprecation_warning("objects = nil",
```

Comment:
`filtered(nil)`

(that said, I'm 90% sure I only ever called this with a collection)